src/components/Input/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef } from "react";
+import { classNames, resetClasses } from "../../util/classes";
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  className?: string;
+}
+
+export const Input = forwardRef<HTMLInputElement, InputProps>(
+  ({ className, ...props }, ref) => {
+    return (
+      <input
+        ref={ref}
+        className={classNames(
+          resetClasses,
+          "ink-w-full ink-rounded-8 ink-bg-background-container ink-p-2 ink-text-body-2",
```

Comment:
I think it's missing `ink-text-text-default`?

Neo:

![image](https://github.com/user-attachments/assets/b04d967e-62a3-40a6-8604-053436df55e3)

Dark:

![image](https://github.com/user-attachments/assets/ea201d4a-7a42-4278-a5ed-7e2f6302e97a)

Reply:
Good spot!